source/slang/slang-diagnostics.lua

```
@@ -122,6 +122,12 @@ local err = helpers.err
 local warning = helpers.warning
 local internal = helpers.internal
 local fatal = helpers.fatal
+-- Warning-level (group) sentinels: pass one positionally to warning() to make that warning
+-- opt-in behind -Wall/-Wextra/-Wpedantic respectively, e.g.
+--   warning("my-warning", 123, "message", span{...}, pedantic)
+local all = helpers.all
```

Luacheck (1.2.0)

[warning] unused variable 'all'

(W211)